package.json

```diff
@@ -1,14 +1,13 @@
 {
   "name": "x11-hash-js",
```

Comment:
What if we released that under @dashevo/x11-hash-js on NPM ?

Reply:
what's that Alex?

Reply:
So basically, right now by pushing it `npm publish` we will publish it here : https://www.npmjs.com/package/x11-hash-js

While we have to do it first (in order to provide last fixes), I suggest that we follow the work done in dashcore-lib for instance, where we deprecated `npm deprecate ...` the original repo in favour of a scopped one. 
See this one : 

Prev : https://www.npmjs.com/package/bitcore-lib-dash 
Now : https://www.npmjs.com/package/@dashevo/dashcore-lib
(do not take attention on the name, however see the `@dashevo` this is the scope, meaning each of our member in @dashevo can update new version of the package, it allow us to avoid bus-factor's risks). 

Npm has a member settings (member, admin, owner) which allow us to still get some permissions control and by the way, you are a owner on this :) ).. 
So I kind of suggest doing the same for this case